benchmark-overhead/Dockerfile-petclinic-base

```diff
@@ -10,6 +10,9 @@ RUN git clone http://github.com/spring-petclinic/spring-petclinic-rest.git
 WORKDIR /app/spring-petclinic-rest
 RUN ./mvnw package -Dmaven.test.skip=true
 RUN cp target/spring-petclinic-rest*.jar /app/spring-petclinic-rest.jar
+
+FROM bellsoft/liberica-openjdk-alpine:11
```

Comment:
I think you can build with Liberica as well. I do :)

Reply:
Maybe, but it doesn't have `git` and I have no idea how to reasonably install packages in there without making it a bunch more complicated.